content/en/archived-docs/corda-os/4.4/deploying-a-node.md

```diff
@@ -196,23 +196,15 @@ at boot, and means the Corda service stays running with no users connected to th
 
 **Prerequisites**:
 
+A supported Java distribution. The supported versions are listed in [Getting set up for CorDapp development](getting-set-up.md)
```

Comment:
```suggestion
A supported Java distribution. The supported versions are listed in [Getting set up for CorDapp development](getting-set-up.md).
```
Punctuation is a bit inconsistent in those steps. I added full stops but feel free to ignore my comments if you want to have the steps without closing full stops. Just review them to make everything consistent please.

Reply:
Done - I'll add full stops where possible - for the lines ending in a URL, I'll move the URL to an indented separate line as the full stop looks odd at the end of it.